src/visualizer/controller.js

```diff
@@ -0,0 +1,117 @@
+(function() {
+    const {
+        getListOfData,
+        getData
+    } = window.comm;
+    const {
+        createGroupInfoElement,
+        createAverageValueTable
+    } = window.detailsElement;
+    const {
+        createGroupSvg
+    } = window.svg;
+
+    getListOfData().then(responds => {
+        console.log(responds);
+        createButtons(responds);
+    });
+
+    /**
+     * @summary Creates a button for each file in the data folder
+     * @param {Array} array this is an array with the names of the files in the data folder
+     */
+    function createButtons(array) {
```

Comment:
brug array.filter((f) => f !== ".gitkeep")
i stedet for det der if i for loopet

Reply:
Fixed

Reply:
Filtrer .gitkeep ud hvor det passer bedre